### DISCUSSION TITLE
Problem to render Quarto Blog, Error Permissiondenied OS Error 5

### DISCUSSION THREAD

**Discussion by MethodenKompetenzZentrum-TUC** [quarto-dev/quarto-cli#7616] [Q&A] [answered]
### Description

I try to setup a Quarto Blog via Rstudio. my system:

R-4.3.2
RStudio-2023.09.1-494
Quarto 1.3.450
Windows10 Enterprise Version 10.0.10240 Build 10240
I freshly installed R, RStudio and Quarto and changed nothing in the configurations. When I try to setup a simple quarto blog in Rstudio (new Project -> quarto blog) and then press the render button, I get the error:

> ERROR: PermissionDenied: Zugriff verweigert (os error 5), stat 'D:\Arbeit\ooo\index_files

I run Rstudio as administrator and checked the security settings of the directory (tried other folders, partitions and even an other PC)

Searched the internet but I found no solutions. Hopefully somebody has an idea

Thanks in advance

**Comment by mcanouil** [comment]
Why did you close the discussion?
Did you solve the issue?

**Reply by MethodenKompetenzZentrum-TUC** [reply]
that was just an mistake, sorry

**Reply by mcanouil** [reply, marked as answer]
Ok.
I am actually going to close this as a duplicate of:
- #5104

Please since the last user that reported this did not follow up, continue on the issue with additional information.

**Reply by MethodenKompetenzZentrum-TUC** [reply]
ok
